src/controllers/auth-controller.ts

```diff
@@ -13,11 +14,15 @@ export default class AuthController extends AbstractController {
 
   private tokens: Repository<RefreshToken>;
 
+  private audit: AuditController;
```

Comment:
When you had created controller didnt it wonder you that one controller is used inside other? May be it's just a service? ))